doc/api/process.md

```diff
@@ -1642,6 +1642,40 @@ This function is only available on POSIX platforms (i.e. not Windows or
 Android).
 This feature is not available in [`Worker`][] threads.
 
+## process.nice(inc)
+<!-- YAML
+added: v10.?.?
```

Comment:
Can you use `REPLACEME` as the placeholder here? It gets picked up by release tooling automatically

Reply:
Done.